### PR TITLE
feature/section-diver

### PR DESCRIPTION
**Pull request by Konmly**
Сreating a diver section.

q1
q2
q3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +48,48 @@ <h3 class="octopus__subtitle">
           </button>
         </div>
       </section>
+      <section class="diver">
+        <div class="diver__cards">
+          <div class="diver__cards1">
```
Comment (PahaTonk) [inline index.html:53]:
```suggestion
          <div class="diver__cards1">
```
Bad name

File: index.html [hunk 2]
```diff
@@ -47,6 +48,48 @@ <h3 class="octopus__subtitle">
           </button>
         </div>
       </section>
+      <section class="diver">
```
Comment (PahaTonk) [inline index.html:51]: Where is container?

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Konmly) [reply index.html:51]: it is not possible to put a container, because then there is no indentation between the columns.

Comment (PahaTonk) [reply index.html:51]: No, it requires a container. It has no effect on the gap between columns

File: styles/diver.css [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.diver__cardone-bkgr {
+    background-color: #523A64;
+    background-image: url("../images/01.png");
+    background-repeat: no-repeat;
+    background-position: top 43px left 0;
+}
+
+.diver__card2-bkgr {
```
Comment (PahaTonk) [inline styles/diver.css:84]: Please do not use numbers in classes

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -47,6 +48,56 @@ <h3 class="octopus__subtitle">
...
+        <div class="diver__cards container">
+          <dl class="diver__cards">
+            <div class="diver__card">
+              <dt class="diver__card-title">
+                <span class="diver__card-numbers">01</span>
```
Comment (PahaTonk) [inline index.html:56]:
```suggestion
                <span class="diver__card-number">01</span>
```

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: styles/diver.css [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+    height: 308px;
+
+    box-shadow:
+    linear-gradient(178.68deg, #4F3963 -2.24%, #764469 31.33%, #5E3E66 76.25%, #2C2142 94.12%);
+    border-radius: 5px;
+    background-color: #523A64;
+}
+
+.diver__card-terminology {
```
Comment (PahaTonk) [inline styles/diver.css:43]:
```suggestion
.diver__card-termin {
```

File: styles/diver.css [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+.diver__card:nth-child(2) {
+    margin-top: 232px;
+}
+
+.diver__card:nth-child(even) {
+    margin-bottom: 160px;
+}
+
+.diver__card:nth-child(2n+3) {
+    margin-top: -75px;
```
Comment (PahaTonk) [inline styles/diver.css:70]: image
```suggestion
    margin-top: -76px;
```

**Review by PahaTonk — CHANGES_REQUESTED** [review]
Please, delete default margins on `dl`
image

**Comment by PahaTonk** [comment]
image
`box-shadow` doesn't work

File: index.html [hunk 7]
```diff
@@ -47,6 +48,54 @@ <h3 class="octopus__subtitle">
           </button>
         </div>
       </section>
+      <section class="diver">
+        <div class="diver__cards container">
```
Comment (PahaTonk) [inline index.html:52]:
```suggestion
        <div class="diver__container container">
            <div class="diver__cards">
```
```suggestion
        <div class="diver__cards container">
```

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: styles/diver.css [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    line-height: 112px;
+}
+
+.diver__text {
+    width: 652px;
+    height: 192px;
+
+    color: #F6F6F6;
+    font-size: 64px;
+    line-height: 150%;
```
Comment (PahaTonk) [inline styles/diver.css:92]: please use a consistent value. Look at line number 83

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: index.html [hunk 9]
```diff
@@ -47,6 +48,64 @@ <h3 class="octopus__subtitle">
...
+      <iframe
+        width="100%"
+        height="668px"
+        src="https://www.youtube.com/embed/KQKCLd_6DGw"
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+      >
+      </iframe>
```
Comment (PahaTonk) [inline index.html:108]:
```suggestion
      </div>
      <div class="diver__video">
        <div class="diver__container container">
          <h2 class="diver__text">Посмотрите видео как работает виджет</h2>
        </div>
      </div>
      <iframe
          width="100%"
          height="668px"
          src="https://www.youtube.com/embed/KQKCLd_6DGw"
          title="YouTube video player"
          frameborder="0"
          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
          allowfullscreen
      >
      </iframe>
  </section>
```

**Review by PahaTonk — CHANGES_REQUESTED** [review]

**Review by PahaTonk — CHANGES_REQUESTED** [review]

**Review by PahaTonk — CHANGES_REQUESTED** [review]

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: index.html [hunk 10]
```diff
@@ -47,6 +48,72 @@ <h3 class="octopus__subtitle">
...
+              </dt>
+              <dd class="diver__card-description">
+                Уже через 28 секунд Ваш сотрудник общается с клиентом и делает
+                продажу!
+              </dd>
+            </div>
+          </dl>
+        </div>
+        <div class="diver__video">
+          <div class="video__container container">
```
Comment (PahaTonk) [inline index.html:98]: Where is the `video` parent block found?

File: styles/diver.css [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+    flex-direction: column;
+
+    color: #F6F6F6;
+    font-size: 64px;
+    line-height: 96px;
+
+    margin-bottom: 70px;
+}
+
+.video__container {
```
Comment (PahaTonk) [inline styles/diver.css:95]:
```suggestion
.diver__video {
```
```suggestion
.video__container {
```

File: styles/styles.css [hunk 12]
```diff
@@ -14,6 +14,10 @@
     src: url("./Roboto-BoldItalic.ttf");
 }
 
+*{
```
Comment (PahaTonk) [inline styles/styles.css:17]:
```suggestion
* {
```

**Review by PahaTonk — APPROVED** [review]

**Pull request merged**